mlx/xunit2rst.mako

```diff
@@ -13,17 +13,19 @@ def _convert_name(name):
     return converted_name
 
 
-def generate_body(input_string):
+def generate_body(input_string, error_type):
     ''' Transforms the input string to be part of an item's indented body with word wrapping.
 
     Args:
-        input_string (str): Raw docstring.
+        input_string (str): Raw error message.
+        error_type (str): The name of the error class. This is 'Error' if it is not specified in the input file.
```

Comment:
Comments and argument doesnt match? I assumed its optional, but in fact it is not and you pass default value as argument. So either expand comment or prepend in here...